internal/target/incus.go

```diff
@@ -341,27 +366,7 @@ func (t *InternalIncusTarget) CreateVMDefinition(instanceDef migration.Instance,
 		numDisks++
 	}
 
-	// Add TPM if needed.
-	if instanceDef.TPMPresent {
-		ret.Devices["vtpm"] = map[string]string{
-			"type": "tpm",
-			"path": "/dev/tpm0",
-		}
-	}
-
-	// Set UEFI and secure boot configs
-	if instanceDef.UseLegacyBios {
-		ret.Config["security.csm"] = "true"
-	} else {
-		ret.Config["security.csm"] = "false"
-	}
-
-	if instanceDef.SecureBootEnabled {
-		ret.Config["security.secureboot"] = "true"
-	} else {
-		ret.Config["security.secureboot"] = "false"
-	}
-
+	ret.Config["security.secureboot"] = "false"
```

Comment:
Might as well set `securit.csm=false` for consistency with post-migration and just in case a default profile forces csm on.

Reply:
Done.